modules/apm/src/main/java/org/elasticsearch/telemetry/apm/APMMeterRegistry.java

```diff
@@ -99,6 +105,16 @@ public DoubleGauge getDoubleGauge(String name) {
         return doubleGauges.get(name);
```

Comment:
Are gauges without a callback actually useful? Could we have this new pattern replace the old for gauges, and add them back if we need them in the future?

Reply:
I think we should remove them.

Reply:
I replaced them with the callback implementation.